images/create-env/Dockerfile

```diff
@@ -1,3 +1,4 @@
+ARG arch=x86_64
 # Use the exact conda, mamba versions as used in bioconda-recipes' builds.
 ARG bioconda_utils_version='1.1.3'
 FROM quay.io/bioconda/bioconda-utils-build-env-cos7:${bioconda_utils_version} as bioconda-build-env
```

Comment:
It seems the image should be `quay.io/bioconda/bioconda-utils-build-env-cos7` for x86 and `quay.io/bioconda/bioconda-utils-build-env-cos7-aarch64` for aarch64?

[1] https://github.com/bioconda/bioconda-utils/pull/866/files#diff-8b50c1c8ac07bc956388add66e256e8336ae22fd2cac8d83f770e1dc63674c00R20-R25

Reply:
Yes, this has to be updated once https://github.com/bioconda/bioconda-utils/pull/866 is merged!